src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,22 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
+        Engine engine = new Engine();
+        engine.setVolume(2.0);
+        engine.setWeight(500);
+        engine.setNumber(1999199);
+        engine.setPower(250);
+        engine.setManufacturer("Bayerische Motoren Werke");
 
+        Car car = new Car();
+        car.setEngineOfCar(engine);
+        car.setMark("BMW");
+        car.setMaxSpeed(250);
+        car.setColor("Black");
+        car.setWeight(2000);
+
+        Car carClone = (Car) car.clone();
+        carClone.getEngineOfCar().setManufacturer("Munich Motoren Werke");
+        System.out.println(car.equals(carClone));
```

Comment:
let's not print out the result of `equals` method, but print each car and see if their engines are different